src/components/pill-selector-dropdown/Pill.js

```diff
@@ -3,19 +3,29 @@ import React from 'react';
 import classNames from 'classnames';
 
 type Props = {
+    isDisabled?: boolean,
     isSelected?: boolean,
     isValid?: boolean,
     onRemove: Function,
     text: string,
 };
 
-const Pill = ({ isSelected = false, isValid = true, onRemove, text }: Props) => (
-    <span className={classNames('pill', { 'is-selected': isSelected, 'is-invalid': !isValid })}>
-        <span className="pill-text">{text}</span>
-        <span aria-hidden="true" className="close-btn" onClick={onRemove}>
-            ✕
+const Pill = ({ isDisabled = false, isSelected = false, isValid = true, onRemove, text }: Props) => {
+    const removeFn = isDisabled ? undefined : onRemove;
+    return (
+        <span
+            className={classNames('pill', {
+                'is-selected': isSelected,
+                'is-invalid': !isValid,
+                'is-disabled': isDisabled,
+            })}
+        >
+            <span className="pill-text">{text}</span>
+            <span aria-hidden="true" className="close-btn" onClick={removeFn}>
```

Comment:
I realize this was already here, but using a span here seems a little sketchy on the semantics side of things.

Reply:
True, I'll update.